tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj

```diff
@@ -7,13 +7,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ML" Version="1.1.0" />
-    <PackageReference Include="Microsoft.ML.DataView" Version="1.1.0" />
-    <PackageReference Include="Microsoft.ML.CpuMath" Version="1.1.0" />
-    <PackageReference Include="Microsoft.ML.FastTree" Version="1.1.0" />
-    <PackageReference Include="Microsoft.ML.LightGbm" Version="1.1.0" />
-    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.1.0" />
-    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="1.1.0" />
+    <PackageReference Include="Microsoft.ML" Version="1.2.0" />
+    <PackageReference Include="Microsoft.ML.DataView" Version="1.2.0" />
+    <PackageReference Include="Microsoft.ML.CpuMath" Version="1.2.0" />
+    <PackageReference Include="Microsoft.ML.FastTree" Version="1.2.0" />
+    <PackageReference Include="Microsoft.ML.LightGbm" Version="1.2.0" />
+    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.2.0" />
+    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="1.2.0" />
```

Comment:
We should also add TensorFlow, TimeSeries, and Onnx, since they are now stable APIs.

Reply:
Done.